src/middleware/redux.ts

```diff
@@ -18,6 +22,26 @@ export type StoreApiWithRedux<
   dispatchFromDevtools: boolean
 }
 
+declare module '../vanilla' {
+  interface StoreMutators<S, A> {
+    'zustand/redux': WithRedux<S, A>
+  }
+}
+
+type WithRedux<S, A> = Write<Cast<S, object>, StoreRedux<Cast<A, Action>>>
+
+interface Action {
+  type: unknown
+}
+
+interface StoreRedux<A extends Action> {
+  dispatch: (a: A) => A
+  dispatchFromDevtools: true
+}
```

Comment:
actually, this one too. okay, i know it's just a preference, but prefer define thing in advance whenever possible. (at least for me it's easier to read in my brain.)

Reply:
No problems, my style is actually opposite of yours haha.